recruitment/introduction.md

```diff
@@ -0,0 +1,26 @@
+## Holiday Extras
+
+#### Why a technical task?
+
+You might be wondering why we feel its important to test candidates. At Holiday Extras we use a variety of different technologies and we don't expect you to be aware of all of them! We've designed both of our tasks with this in mind.
+
+Below is a few points that we tend to look for when reviewing your submission:
+
+- Quality (is it functional?)
+- Structure (code layout, abstraction, modularisation)
+- Usability and design
+
+#### The tasks
+
+- [Build an API](developer-API-task.md)
+- [Flickr API](developer-flickr-task.md)
+
+If you have _not_ been asked to complete a specific task in the email you received from our recruitment team then we're leaving it up to you! Feel free to pick whichever task you believe would best showcase your skills.
```

Comment:
I am happy with how this now is. I feel we have said we may tell you which task to do, or we may let the candidate choose.

@StefKennett for the Full Stack role, always push people towards the Flickr test. If the candidate responds that they are not comfortable with that test then we can review. If in doubt go with Flickr test ;)

I think we could try to push to a "Main or Primary" test, but lets merge this and see what feedback we get. These docs can be iterative, once 5+ people have gone through this we will probably do another minor PR to make small tweaks.